src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyFactory.java

```diff
@@ -118,13 +119,15 @@ public HttpCacheProxyFactory(
         LongSupplier supplyTrace,
         ToIntFunction<String> supplyTypeId,
         SignalingExecutor executor,
-        LongObjectBiConsumer<Runnable> scheduler)
+        LongObjectBiConsumer<Runnable> scheduler,
+        int preferWaitMaximum)
     {
         this.router = requireNonNull(router);
         this.budgetManager = requireNonNull(budgetManager);
         this.supplyInitialId = requireNonNull(supplyInitialId);
         this.supplyTrace = requireNonNull(supplyTrace);
         this.supplyReplyId = requireNonNull(supplyReplyId);
+        this.preferWaitMaximum = preferWaitMaximum;
```

Comment:
This becomes `this.preferWaitMaximum = config.preferWaitMaximum();`

Reply:
Done.